clearwater-diags-monitor/usr/share/clearwater/bin/clearwater_diags_monitor

```diff
@@ -435,6 +460,24 @@ do
     inotifywait -e create -qq .
   fi
 
+  # Now wait for CPU load to be at an acceptable level.  Each time it isn't,
+  # check for new files in the crash directory and clear out the newest ones
+  # until we're not using more than 1GB - we don't want to run out of disk
+  # space.  Pause for 20s before we start, just so that any previous process
+  # has an opportunity to restart first.
+  sleep 20
+  while [ $(idle_cpu) -lt $MIN_IDLE_CPU_FOR_GATHER ]
+  do
+    log "CPU usage too high - not gathering diagnostics yet"
+    while [ $(disk_usage_in_kb .) -gt $ONE_GB_IN_KB ]
+    do
+      trigger_to_delete=$(ls -At | head -n 1)
```

Comment:
As just discussed, this is the trigger file we're deleting, not the capture file.  I think it's right to delete all of these if they're actually more than 1GB.  (We haven't yet done any work on them, so we're not throwing much away.)